src/qml/CoordinateLocator.qml

```diff
@@ -48,15 +49,49 @@ Item {
     return !!mapSettings.visibleExtent || true ? mapSettings.coordinateToScreen(currentCoordinate) : 0
   }
 
+  property variant rubberbandModel: null;
```

Comment:
null or undefined? also, whether we decide to add semi columns or not to the javascript code, let's not do it for property values :)

Reply:
I am used to use `null` instead of QML undefined. Fixed, also removed the `;`. Thanks for spotting.